tests/test_scars.py

```diff
@@ -0,0 +1,17 @@
+"""Tests for the scars module."""
+from pathlib import Path
+import numpy as np
+
+from topostats.scars import Scars
+
+BASE_DIR = Path.cwd()
+RESOURCES = BASE_DIR / "tests" / "resources"
+
+
+def test_remove_scars(synthetic_scar_removal: Scars):
```

Comment:
This is akin to a regression test since it's testing the overall result.

Unit-tests test the individual methods for their action under different conditions and it would be useful to have unit tests for the different methods in place.

Reply:
Added more tests 👍